zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -180,11 +215,15 @@ def rollback_snapshot(logical_pool, volume, snapshot):
 
 
 def cbd_to_nbd(desc, port, install_path):
-    logger.debug("qemu-nbd -D %s -f raw -p %d --fork %s_%s_:%s" % (desc, port, install_path, ZBS_USER_NAME, ZBS_CLIENT_CONF_PATH))
-    os.system("qemu-nbd -D %s -f raw -p %d --fork %s_%s_:%s" % (desc, port, install_path, ZBS_USER_NAME, ZBS_CLIENT_CONF_PATH))
+    cmd = "qemu-nbd -D %s -f raw -p %d --fork %s_%s_:%s" % (
+        desc, port, install_path, ZBS_USER_NAME, ZBS_CLIENT_CONF_PATH)
+    logger.debug(cmd)
+    os.system(cmd)
```

Ruff (0.13.3)

Starting a process with a shell, possible injection detected

(S605)